src/openrct2/windows/map.c

```diff
@@ -176,7 +176,7 @@ static uint8 _activeTool;
 static uint32 _currentLine;
 
 /** rct2: 0x00F1AD68 */
-static uint8 (*_mapImageData)[512][512];
```

Comment:
Would it be an idea to put a comment in explaining Marijn's point and leave @janisozaur 's code as it is?